src/components/10-atoms/link/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.0.0
+
+- BREAKING CHANGE: Now the referenced icons have dimensions 24x24 pixels which is set by `icons` component. (#1725)
```

Comment:
"Now the referenced icons have the dimension 24x24 pixels, which is set by the `icons` component."